onboardingtaskapp.client/src/components/Customer/Edit.jsx

```diff
@@ -55,20 +69,32 @@ export default function Edit({ item }) {
             }
             // Handle success
             console.log('Updated successfully');
```

Comment:
Can you remove this? We need to remove console.log in product level so users won't be able to see any unwated data. Thanks